games/TicTacToe/project/tictactoe-contract/Forc.toml

```
@@ -0,0 +1,7 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "TicTacToe"
```

```suggestion
name = "tictactoe-contract"
```
Applying this suggestion would require changing the paths in `tests/utils/setup.rs`